src/Uno.Foundation/Uno.Core.Extensions/Uno.Core.Extensions/StringExtensions.Format.cs

```diff
@@ -1,248 +0,0 @@
-// ******************************************************************
-// Copyright � 2015-2018 nventive inc. All rights reserved.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-//
-// ******************************************************************
-using System;
-using System.Collections.Generic;
-using System.Globalization;
-using System.Linq;
-using System.Text;
-
-namespace Uno.Extensions
-{
-	internal static partial class StringExtensions
```

Comment:
This whole class is looking unused. So I decided to delete it instead of fixing warnings in it.